src/dag/read.rs

```diff
@@ -54,3 +54,39 @@ pub async fn get_head(kvr: &dyn kv::Read, name: &str) -> Result<Option<String>>
     }
     Ok(None)
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::kv::memstore::MemStore;
+    use crate::kv::Store;
+
+    #[async_std::test]
+    async fn test_has_chunk() {
+        async fn test(hash: &str, del_haskey: bool, expect_has: bool) {
```

Comment:
I find the `has` part of this `del_haskey` name sort of confusing. Maybe `delete` or `delete_key`?

Reply:
done in https://github.com/rocicorp/repc/pull/41